### ISSUE TITLE
Missing ALT tag PPT

### ISSUE THREAD

**Issue by dzaremba90** [Microsoft/MCW-Containers-and-DevOps#13]
This image is missing the alt text in the ppt slide. 

![image](https://user-images.githubusercontent.com/40214167/42105484-54f02fc8-7b85-11e8-88de-cb9f89deefef.png)

**Comment by michelebusta** [comment]
alt text all fixed
